### ISSUE TITLE
Nokia 5/Android 9: Background apps get optimized if on external storage

### ISSUE THREAD

**Issue by genodeftest** [urbandroid-team/dont-kill-my-app#179]
I'm not entirely sure about this this, so I'm leaving it here as an issue and not as a pull request. I'm also not sure of the scope, i.e. which Android versions and flavors it affects. As it is on a Nokia device which are known to be almost-stock Android, I would guess it *should* affect Android in general.

## Observation 1: NetGuard
I'm using @M66B's great [NetGuard](https://www.netguard.me/) transparent non-root firewall. After I moved it to my external SD card (*Settings*→*Apps & notifications*→*NetGuard*→*Storage*→*Change*), every time it started, it complained about being battery optimized (which may break it). And in fact, Android's settings (*Settings*→*Apps & notifications*→*NetGuard*→*Advanced*→*Battery*) say *Optimizing battery usage*. I was able to change this to *Not optimized*, but this changed back to *Optimizing battery usage* every now and then (reliably after each reboot). After moving the App to the internal storage, disabling battery optimizations persisted even across reboots.

## Observation by others: simpleMobileTools
I'm also using @tibbi's great [SimpleMobileTools](simplemobiletools.github.io/). After moving a bunch of apps from my internal to external storage, one of it (probably the calendar app) complained that its widgets cannot work correctly if installed on external storage. And in fact, the widget was kinda broken. After moving it back to the internal storage, it worked fine.
He added a bit of code in [451b223dd835a819c9d976efb96781b45901588a](https://github.com/SimpleMobileTools/Simple-Commons/commit/451b223dd835a819c9d976efb96781b45901588a#diff-5748e1ce15bbb39966fd2a6f4ee40591)

In other words: You may want to check and add this valuable information to your website.
